supertokens_python/recipe/emailverification/interfaces.py

```diff
@@ -161,24 +164,55 @@ def __init__(self):
 
     @abstractmethod
     async def email_verify_post(
-        self, token: str, api_options: APIOptions, user_context: Dict[str, Any]
+        self,
+        token: str,
+        api_options: APIOptions,
+        user_context: Dict[str, Any],
+        session: Optional[SessionContainer] = None,
```

Comment:
Add session before user_context. And there is no default value for session, but it is optional.

Reply:
Done.